wp/wp-content/themes/lahma_maker/assets/playerfiles/radio_player.vue

```diff
@@ -245,6 +245,26 @@ export default {
         "show_subtitle": function() {
             return this.np.now_playing.song.title
         },
+        "is_OFF_AIR": function() {
+            if (this.np.now_playing.playlist == 'OFF AIR' || this.np.now_playing.playlist !== 'Off Air Ambient') {
+              return true;
+            } else {
+              return false;
+            }
+        },
+        "OFF_AIR_type": function() {
```

Comment:
why is this check? (also the other `_type`)
btw. if the playlist is stored in a computed var, it could be used at `is_` checks also, 
`this. OFF_AIR_type === 'OFF AIR'`